INSTALL.md

```diff
@@ -165,3 +165,17 @@ This option should **not** be used if there are any changes to `.go` files, CI s
 
 In case there are known CI failures that are blocking a merge, project maintainers can add this flag
 to unblock the CI and merge the PR. This should be used with extreme care so regressions are not introduced.
+
+### [test storage-provider]
```

Comment:
I think we should include a warning here similar to this one for `[skip ci]` "This option should **not** be used if there are any changes to `.go` files, CI scripts, etc."

Maybe something like "These options should **not** be used if there are any changes to code which is shared with other backends or code that might possibly be used with other backends."